tests/typ/visualize/polygon.typ

```diff
@@ -8,6 +8,7 @@
 #polygon()
 #polygon((0em, 0pt))
 #polygon((0pt, 0pt), (10pt, 0pt))
+#polygon.regular(size: 0pt, vertices: 9)
```

Comment:
more tests please (e.g. the regular polygon sheet you shared in the PR description)

Reply:
I added a test similar to the PR description.

Reply:
Thanks!